src/viur/core/bones/relational.py

```diff
@@ -619,7 +611,7 @@ def postSavedHandler(self, skel, boneName, key):
             dbObj["viur_dest_kind"] = self.kind
             dbObj["viur_relational_updateLevel"] = self.updateLevel.value
             dbObj["viur_relational_consistency"] = self.consistency.value
-            dbObj["viur_foreign_keys"] = self.refKeys
+            dbObj["viur_foreign_keys"] = tuple(self.refKeys)
```

Comment:
Due https://github.com/viur-framework/viur-datastore/issues/46, we need to use `list()` here.
```suggestion
                dbObj["viur_foreign_keys"] = list(self.refKeys)
```